src/lightning_app/CHANGELOG.md

```diff
@@ -0,0 +1,17 @@
+# Changelog
+
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
+
+## \[0.5.2\] - 2022-MM-DD
```

Comment:
I don't think we have those in the other CHANGELOG

```suggestion
## [0.5.2] - 2022-MM-DD
```

And isn't next release 0.6?

Reply:
We could do both. This hasn't been decided. I would prefer weekly releases.